pkg/version/channels.go

```diff
@@ -14,12 +15,14 @@ import (
 // This module is also responsible for online retrieval of the latest release
 // versions.
 type Channels struct {
-	array []channelVersion
+	Array []channelVersion
```

Comment:
rather than making `array` public, what do you think of adding a public method called `Empty()` which returns `len(c.array) == 0`?